txpool/txpool_test.go

```diff
@@ -70,17 +65,30 @@ func newTx(addr types.Address, nonce, slots uint64) *types.Transaction {
 }
 
 // returns a new txpool with default test config
-func newTestPool(header *types.Header) (*TxPool, error) {
+func newTestPool(mockStore ...store) (*TxPool, error) {
+	return newTestPoolWithSlots(defaultMaxSlots, mockStore...)
+}
```

Comment:
Why the variable `store` parameter ?

Reply:
Because we introduced the ability to specify custom mock stores that the TxPool will use in an earlier PR.

This spread trick just cleans up the interface, so not every method that calls `newTestPool` needs to pass in nil for the store (to signalize a default one should be used)